### DISCUSSION TITLE
Automatic download of remotior-sensus

### DISCUSSION THREAD

**Discussion by chamoisrouge** [semiautomaticgit/SemiAutomaticClassificationPlugin#382] [General]
Dear SCP team,

I use SCP to teach my students. It's the first remote sensing tool they've seen, and they're beginners. Since the switch to remotior-sensus, installing this module has made using SCP extremely complicated, especially on computers where we don't always have admin rights. However, I see that in the code, by uncommenting lines 90 to 165 of the ‘semiautomaticclassificationplugin.py’ file, remotior-sensus is downloaded automatically, which is very practical. **Would it be possible to uncomment these lines in the code provided?** If not, I'm afraid the manual installation of remotior-sensus will be a major obstacle to the use of SCP by the community.
In any case, many thanks for the hard work that went into developing this great tool!

All the best

**Comment by semiautomaticgit** [comment]
Dear @chamoisrouge ,

thank you for using the SCP in your course.

I understand the issues in installing the required dependencies.
Those code lines were commented because there is a more convenient way to install remotior-sensus (without admin rights) following this guide https://semiautomaticclassificationmanual.readthedocs.io/en/latest/faq.html#alternative-installation-of-plugin-dependencies

I hope this solves your installation issues.

**Comment by chamoisrouge** [comment]
Dear @semiautomaticgit ,

Thank you for this prompt answer. It works but I need to open QGIS, install qpip, close QGIS, reopen QGIS and finally install SCP. Normally it's not a big deal but with students everything becomes a bid deal ;) Especially in my specific case, where the computers of the teaching lab are 'reset' after each reboot.
I recently explored another plugin which requires Pyhon packages and it installs them without the use of Qpip. This plugin is "Deepness", and I think the automatic download of necessary packages is done in this part of their code :
https://github.com/PUTvision/qgis-plugin-deepness/blob/devel/src/deepness/dialogs/packages_installer/packages_installer_dialog.py
Do you think it could be an option to think about for SCP?

All the best
